frontend/src/components/DisableableSelectInput.vue

```diff
@@ -0,0 +1,75 @@
+<!--
+ Copyright (c) 2023 Volkswagen AG
+ Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
Please change the copyright year for new and all edited files to the current year (`2024`).

Reply:
Updated existing files to "old year, new year" and new files to "new year"